syft/execution/placeholder.py

```diff
@@ -25,6 +25,17 @@ def __init__(self, owner=None, id=None, tags: set = None, description: str = Non
             self.id = PlaceholderId(self.id)
         self.child = None
 
+    def __getattribute__(self, name):
```

Comment:
@karlhigley moved the logic here.
We also need tome attributes from Placeholder (like the ```id```, ```tags```, ```instantiate``` method, etc.).
and for some of them, we forward them to the child.

Another solution for this would be to use a list of attributes that we know are called on the placeholder.
This list should be kept with the attributes like: ```id```, ```tags``` ... (all the parameters in the constructor) + the methods we define in the Placeholder.

What would look like then:
```
def __getattribute__(self, name):
    if name in ["id", "tags", ....]:
       return object.__getattribute__(self, name)
   else:
       child = object.__getattribute__(self, "child")
       return getattr(child, name)
```

Reply:
I think you could maybe use `hasattr` to check if Placeholder defines `name` and delegate to the child otherwise.

Reply:
The suggestion above doesn't work, because `hasattr` calls `__getattribute__`. Oops!